docs/howto/add-coc.md

```diff
@@ -0,0 +1,4 @@
+## Adding a Code of Conduct to your repository
+
+To include a Code of Conduct (CoC) in your ontology repository, first create a dedicated file named `CODE_OF_CONDUCT.md` in the root directory of your repository. This file should clearly articulate the community's expected behaviors, identify unacceptable conduct such as harassment or discrimination, and describe the mechanisms through which community members can confidentially report any violations. You may base your document on widely recognized templates such as the [Contributor Covenant](https://www.contributor-covenant.org/) or established examples like the [OBO Foundry's Code of Conduct](https://obofoundry.org/code-of-conduct/). Once created, prominently link your CoC from the repository's `README.md`, contribution guidelines (`CONTRIBUTING.md`), and other prominent documentation pages to ensure it is easily discoverable.
```

Comment:
I would add somewhere that the CoC must be tailored to the project it is applied to (especially, the _size_ of the project).

For example, there would be no point in having a CoC that advises to report violations to a “conflict resolution team” if the project itself is a two-person show. This kind of things is only suitable for projects whose community has reached a certain size.

So, don’t pick just _any_ CoC without thinking. Pick a CoC that works for your project.

Reply:
Check above.